nativelink-scheduler/src/simple_scheduler_state_manager.rs

```diff
@@ -629,6 +629,8 @@ where
                     }
                 }
                 UpdateOperationType::UpdateWithDisconnect => ActionStage::Queued,
+                // We shouldn't get here, but we just ignore it if we do.
+                UpdateOperationType::ExecutionComplete => return Ok(()),
```

Comment:
Possibly worth logging, given it shouldn't occur, so we can debug and figure out why later